securedrop/requirements/python3/translation-requirements.txt

```diff
@@ -4,9 +4,14 @@
 #
 #    pip-compile --allow-unsafe --generate-hashes --output-file=requirements/python3/translation-requirements.txt requirements/python3/translation-requirements.in
 #
-babel==2.9.1 \
-    --hash=sha256:ab49e12b91d937cd11f0b67cb259a57ab4ad2b59ac7a3b41d6c06c0ac5b0def9 \
-    --hash=sha256:bc0c176f9f6a994582230df350aa6e05ba2ebe4b3ac317eab29d9be5d2768da0
+babel==2.12.1 \
```

Comment:
Did this get a diff review?

Reply:
Good catch; thanks.  #6996 has the documentation fix; I'll do the diff reviews this afternoon.

Reply:
Done in <https://github.com/freedomofpress/securedrop-builder/wiki/Babel-2.9.1-to-2.12.1>.